ground/src/main/java/com/google/android/ground/persistence/sync/PhotoSyncWorkManager.kt

```diff
@@ -34,7 +34,7 @@ constructor(
   override val workerClass: Class<PhotoSyncWorker>
     get() = PhotoSyncWorker::class.java
 
-  override fun preferredNetworkType(): NetworkType =
+  override val preferredNetworkType: NetworkType =
```

Comment:
This will be frozen to UNMETERED or CONNECTED depending on the value of the localValueStore when the worker is first instantiated--is that what we want?

If not, use a `get()` so that this is instead computed every time we reference the field (then we'd always fetch the latest value from the localValueStore whenever this is referenced).

Reply:
thanks for catching this! reverted this change